doc/devel/development.md

```diff
@@ -0,0 +1,43 @@
+# Developer guide 
```

Comment:
a minor styling issue: put an empty line between headlines and its content. Thanks!